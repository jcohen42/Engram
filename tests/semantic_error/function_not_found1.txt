Begin program:
	Set x equal to 10.
	Call World with argument x.
End program.