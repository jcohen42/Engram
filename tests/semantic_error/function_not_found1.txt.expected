Error: Function World, which was called in line 3, was not found.
Please make sure that this function exists.